### ISSUE TITLE
When editing a demonstration, disable/hide student selector and "continue with next student"

### ISSUE THREAD

**Issue by ckunkel** [SlateFoundation/slate-cbl#428]
When editing a demonstration, the concept of continuing to the next student does not apply. This button should not be present. Because it is, it results in errors presented after the edit is submitted because it is trying to copy over the information to a new demonstration.  

![screen shot 2018-02-09 at 11 51 56 am](https://user-images.githubusercontent.com/5123794/36039174-bd561e2e-0d8f-11e8-8b2b-9db58d2e6117.png)

Additionally, the student selector is usable but navigating to another student does not make sense when you're editing a single demonstration. There is no other demonstration to go to when you select a different student, so this field should be deactivated. 

To summarize, 
1. Deactivate student selector
2. Deactivate "Continue with next student"
